cmd/docker-driver/README.md

```diff
@@ -129,6 +132,8 @@ To specify additional logging driver options, you can use the --log-opt NAME=VAL
 | `loki-tls-server-name`          | No        |                      | Name used to validate the server certificate.
 | `loki-tls-insecure-skip-verify`           | No        | `false`                | Allow to skip tls verification.
 | `loki-proxy-url`                       | No        |           | Proxy URL use to connect to Loki.
+| `max-size`                       | No        |       -1    | The maximum size of the log before it is rolled. A positive integer plus a modifier representing the unit of measure (k, m, or g). Defaults to -1 (unlimited). This is used by json-log required to keep the `docker log` command working.
```

Comment:
should we default to unlimited? I would think something like 100m might be a better default, since most people are not going to want unlimited file growth a default of unlimited would mean everyone needs to configure this flag?

Reply:
This is the default for docker driver. https://docs.docker.com/config/containers/logging/json-file/

Reply:
Though I'm not against doing a better job.

Reply:
I think it makes sense to match the defaults of docker to remain consistent, although I don't feel strongly about this one way or the other